presto-spi/src/main/java/com/facebook/presto/spi/block/FixedWidthBlockBuilder.java

```diff
@@ -92,7 +89,7 @@ public boolean isFull()
     @Override
     public int getSizeInBytes()
     {
-        long size = getRawSlice().length() + SizeOf.sizeOf(valueIsNull);
+        long size = getRawSlice().length() + valueIsNull.getUnderlyingSlice().length();
```

Comment:
No, getRawSlice() gets the underlying slice of the sliceOutput for the data. I'm doing the same for valueIsNull (which is now a SliceOutput)